index.html

```diff
@@ -51,152 +51,174 @@ <h3 class="contact">LET'S CONNECT</h3>
+                        </p>
+                        <div class="project-button">
+                            <ul>
+                                <li><button type="button">html</button></li>
+                                <li><button type="button">css</button></li>
+                                <li><button type="button">JavaScript</button></li>
+                            </ul>
+                        </div>
+                        <div class="see-project">
+                            <button type="button">See project</button>
+                        </div>
                     </div>
                 </div>
             </div>
             <div class="project" id="p4">
                 <div class="project4">
                     <img class="profile_snapshoot" src="./image/SnapshootPortfolio4.png" alt="Profile Snapshoot">
-                    <h2 class="project1-h1">Multi-Post Stories</h2> 
-                    <ul class="project1-ul">
-                        <li id="canopy4">CANOPY</li>
-                        <li>Back End Dev</li>
-                        <li>2022</li>
-                    </ul>
-                    <p class="project1-para">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-                    <div class="project-button">
-                        <ul>
-                            <li><button type="button">html</button></li>
-                            <li><button type="button">css</button></li>
-                            <li><button type="button">JavaScript</button></li>
+                    
+                    <div class="project-content">
+                        <h2 class="project1-h1">Multi-Post Stories</h2> 
+                        <ul class="project1-ul">
+                            <li id="canopy4">CANOPY</li>
+                            <li>Back End Dev</li>
+                            <li>2022</li>
                         </ul>
-                    </div>
-                    <div class="see-project">
-                        <button type="button">See project</button>
+                        <p class="project1-para">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <div class="project-button">
+                            <ul>
+                                <li><button type="button">html</button></li>
+                                <li><button type="button">css</button></li>
+                                <li><button type="button">JavaScript</button></li>
+                            </ul>
+                        </div>
+                        <div class="see-project">
+                            <button type="button">See project</button>
+                        </div>
```

Comment:
- [ ] Great job 👍 . Kindly make sure your buttons have the necessary interactions as seen in the Figma design 🙏 .
This would let a user know when they hover or click a button and it is great practise.

![image](https://user-images.githubusercontent.com/74814780/179040358-0e35e21e-cf47-44ab-85dd-ce9cfd45a876.png)

Reply:
Added the button effect.

Reply:
- [x] This issue is well resolved :heavy_check_mark: